cni/network/network.go

```diff
@@ -742,6 +764,10 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 	}
 
 	if opt.resultV6 != nil {
+		// get IPAM mode from conflist file and add ipv6 routes to linux pod if needed
+		epInfo.IPV6Mode = string(util.IpamMode(opt.nwCfg.IPAM.Mode))
```

Comment:
I think you should only set the epInfo.IPV6Mode to that value if the string(util.IpamMode(opt.nwCfg.IPAM.Mode)) is dualStackOverlay. This will prevent setting the IPV6Mode if we are not operating in dualstack mode